provision/acc_provision/acc_provision.py

```diff
@@ -1480,6 +1480,19 @@ def generate_kube_yaml(args, config, operator_output, operator_tar, operator_cr_
             network_operator_CR_output = network_operator_CR_template.render(config=netopConfig)
             network_operator_yaml = network_operator_spec_output + "\n---\n" + network_operator_CR_output
 
+            # The next few files are to generate tar file with each
+            # containers and operator yaml in separate file. This is needed
+            # by OpenShift >= 4.3. If tar_path is provided(-z), we save the tar
+            # with that filename, else we use the provided containers
+            # deployment filepath. If neither is provided, we don't generate
+            # the tar.
+            if "openshift" in config["flavor"]:
+                if tar_path == "-":
+                    tar_path = "/dev/null"
+                else:
+                    deployment_docs = yaml.load_all(network_operator_yaml, Loader=yaml.SafeLoader)
+                    generate_operator_tar(tar_path, deployment_docs, config)
```

Comment:
`
noiro@oshift2-ext-rtr:~$ acc-provision --cko  -c acc_provision_cko_input.yaml -f openshift-4.10-esx -u admin -p noir0123 -o netop-manager-cko-testing.yaml -z netop-manager.tar.gz --debug
INFO: Loading configuration from "acc_provision_cko_input.yaml"
INFO: Using configuration flavor openshift-4.10-esx
INFO: Reusing existing certs for kubernetes controller
INFO:   Private key file: "user.key"
INFO:   Certificate file: "user.crt"
writing the deployment file
...
tar -xvf netop-manager.tar.gz -C netoptar/
cluster-network-04-Namespace-netop-manager-system.yaml
cluster-network-05-CustomResourceDefinition-cniops.cni-manager.netop-manager.io.yaml
cluster-network-06-CustomResourceDefinition-installers.netop-manager-cntrlr.netop-manager.io.yaml
cluster-network-07-ServiceAccount-netop-manager-controller-manager.yaml
cluster-network-08-Role-netop-manager-leader-election-role.yaml
cluster-network-09-ClusterRole-netop-manager-manager-role.yaml
cluster-network-10-ClusterRole-netop-manager-metrics-reader.yaml
cluster-network-11-ClusterRole-netop-manager-proxy-role.yaml
cluster-network-12-RoleBinding-netop-manager-leader-election-rolebinding.yaml
cluster-network-13-ClusterRoleBinding-netop-manager-manager-rolebinding.yaml
cluster-network-14-ClusterRoleBinding-netop-manager-proxy-rolebinding.yaml
cluster-network-15-ConfigMap-netop-manager-cni-config.yaml
cluster-network-16-ConfigMap-netop-manager-config.yaml
cluster-network-17-Service-netop-manager-controller-manager-metrics-service.yaml
cluster-network-18-Deployment-netop-manager-controller-manager.yaml
cluster-network-19-ConfigMap-cni-config.yaml
cluster-network-20-ConfigMap-status-config.yaml
cluster-network-21-PriorityClass-netop-priority.yaml
cluster-network-22-Installer-installer-aci-cni.yaml
cluster-network-03-config.yaml
apic.json
`

Reply:
I believe the names can be shortened like how it is done in categories
CRDs
1.  cluster-network-<num>-<Groups>.yaml
ConfiMap
3.  cluster-network-<num>-ConfiMap-<name>.yaml
ClusterRole/Binding/Deployment/Daemonset
5. cluster-network-<num>-<Kind>-name.yaml

Take the reference for current nomenclature how it is in aci.tar